### ISSUE TITLE
When I tried to drag a phrase to another folder I lost all of my entries (all folders, all entries)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eadevlin/autokey#119]
```
Am too scared to try to replicate the problem.
```

Original issue reported on code.google.com by `elmar.ro...@gmail.com` on 2 Jun 2011 at 2:15

**Comment by GoogleCodeExporter** [comment]
```
You haven't filled out the bug form as suggested. We need at least to know what 
version of AutoKey and Linux you are running.
```

Original comment by `cdekter` on 2 Jun 2011 at 11:21
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
This was caused by a behaviour change in GTK and is fixed in AutoKey 0.80.0
```

Original comment by `cdekter` on 19 Sep 2011 at 12:33
- Changed state: **Fixed**
